src/components/ProgressBar.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import styled from 'styled-components'
+
+const ProgressBarContainer = styled.div`
+  color: ${p => p.theme.colors.text};
+  display: flex;
+  height: 15px;
+  margin: 1em;
+  border-radius: 3px;
+  overflow: hidden;
+  border: 1px solid ${p => p.theme.colors.primary};
+`
+
+// filled part of progress bar
+const FlexPrimary = styled.div`
+  flex: 0 0 ${p => p.percent * 100}%;
+  background-color: ${p => p.theme.colors.highlight};
+`
+
+// rest of progress bar
+const FlexSecondary = styled.div`
+  background-color: ${p => p.theme.colors.primary};
+`
+
+const ProgressBar = ({current, total}) => {
```

Comment:
do we want to just have progress bar take a percent? Or a value out of 100?

Reply:
good idea, that way it would make sense for datetime objects too and we can handle that percentage calculation in the container